### PR TITLE
test: Run check-machines on more systems

### PR DESCRIPTION
**Pull request by stefwalter**
First of all it's far too easy to ignore the check-machines test
and skip it. This shouldn't be the case. The test machines should
have nested virtualization configured.

Secondly unloading and loading the kvm-intel kernel module in
the kernel causes a race with udev, and ends up with messages
like this:

Could not access KVM kernel module: Permission denied
failed to initialize KVM: Permission denied

This is due to rules in /usr/lib/udev/rules.d which set the
permissions not having run yet.

### CODE REVIEW

File: test/vm-prep [hunk 1]
```diff
@@ -79,6 +79,12 @@ EOF
         echo "$rule" >> ${qemu_config_dir}/bridge.conf
       fi
   done
+
+    # We really want nested virtualization
+    if [ ! -f /etc/modprobe.d/kvm-intel.conf ]; then
```
Comment (dperpeet) [inline test/vm-prep]: This assumes that all test runners have `kvm_intel` and not `kvm_amd`. Also, `vm-prep` is intended to prepare all tests, but the verify ones are only a subset. Should we really impose these two constraints for all test runners?

Comment (stefwalter) [inline test/vm-prep]: It's not a constraint. Failures are ignored.

But i'll add support for AMD.

Comment (stefwalter) [inline test/vm-prep]: Yup. The naming of the files is irrelevant.

**Comment by stefwalter** [comment]
cockpit-machines needs to be built on Debian.

**Comment by stefwalter** [comment]
This should now also fix check-machines on Debian.

**Comment by dperpeet** [comment]
no `virt-install` available in our Debian image

**Comment by stefwalter** [comment]
Added virt-inst to debian-unstable image.

**Comment by stefwalter** [comment]
Enabling net-autostart for default libvirt network on Debian....

**Pull request merged**